packages/react/src/MinimalFooter/MinimalFooter.module.css

```diff
@@ -59,6 +59,11 @@
   padding-bottom: var(--base-size-48);
 }
 
+.Footer__social-link:hover .Footer__social-icon,
+.Footer__social-link:focus-visible .Footer__social-icon {
+  filter: var(--brand-footer-socialIcon-hoverFilter);
```

Comment:
Should there be any transition between the idle and active states @danielguillan? Feels a bit sudden currently, but non-blocking feedback / question.

Reply:
Definitely! We can add those in a follow-up PR.